ckanext/ontario_theme/templates/internal/home/layout3.html

```diff
@@ -7,12 +7,17 @@
     </div>
     <div>
       {% snippet "home/snippets/ontario_theme_sub_intro.html" %}
+      {# Conditionally show Topics: hide on intra.data.ontario.ca #}
+      {% if request and request.host and request.host.split(':')[0] == 'intra.data.ontario.ca' %}
+        {# Hidden on IDC #}
+      {% else %}
       <div id="topics">
```

Comment:
can you also indent the `else` block so it follows the styling convention.
you can also use djlint for fixing styling.